library/Fission/User/Model

```diff
@@ -0,0 +1,13 @@
+User
+  active Bool
+  email Text Maybe
+  herokuAddOnId HerokuAddonId Maybe
+  role Role
+  secretDigest Text
+  username Text
+
+  insertedAt UTCTime
+  modifiedAt UTCTime
+
+  UniqueSecretDigest secretDigest
+  UniqueUsername username
```

Comment:
Oh nice! I didn't realize that you could do it this way 🎉 I'd only veer seen the "one big file" approach. Does this still enforce FKs and whatnot?

Also, how do you feel about calling these like `Model.entity`, or `Model.persistent`, or... something? Just feels weird without a file extension

Reply:
It should enforce the FKs yeah. There's some links in the collapsed examples section in the [docs](https://www.stackage.org/haddock/lts-14.4/persistent-template-2.6.0/Database-Persist-TH.html#v:persistManyFileWith).

Will rename to `Model.entity` 👍